docs/cli/server.md

```diff
@@ -761,14 +673,6 @@ Disable automatic session expiry bumping due to activity. This forces all sessio
 
 Disable password authentication. This is recommended for security purposes in production deployments that rely on an identity provider. Any user with the owner role will be able to sign in with their password regardless of this setting to avoid potential lock out. If you are locked out of your account, you can use the `coder server create-admin` command to create a new admin user directly in the database.
 
-### --config, -c
```

Comment:
Where all these flags supposed to disappear?

Reply:
Yep. They're hidden as the YAML code is untested.